### ISSUE TITLE
Generate simulated training and test dataset

### ISSUE THREAD

**Issue by rcurrie** [braingeneers/braingeneers#3]
Generate simulated data with inputs and outputs as a proxy for future MEA data. Training data should provide labels as to which outputs are connected to inputs. Test data should not. Nominal idea is to wrangle this into PRP CEPH and run DeepCorr against it to see if it can learn the correlations and predict the test sets input/output pairs.

**Comment by rcurrie** [comment]
Following up on the experiment to use SHAP to stain the forthcoming experimental images: @ashsrobbins @kvoitiuk Is it a valid assumption that the input vector itself (whether light, electrode etc...) will not be directly present in the output image if we select an image that's delayed appropriately from the application of the stimulus? This is important so that we don't end up training an image classifier on just the input essentially. 

@atspaeth Can you generate tuples of input vector and output image whereby the output image does NOT have any of the input present? If so go ahead and generate a few just so I can sort out what the input format is (whatever is easiest - compressed image fine or numpy). I can try and get a classifier + SHAP working on it as an exploration.

**Comment by rcurrie** [comment]
Excellent - if easy leave off the ticks although it won't make any difference to the code really. Inputs will be multi-label or one at a time?

**Comment by atspaeth** [comment]
I'm about halfway through generating 1000 examples as png files, with names formatted like printf("eg%04d_%d.png", example_number, label). There is exactly one input active in every image; that's what the label refers to. 

Now that stimulation ends before taking the picture, the images aren't as obvious to classify by eye, but I think there's still enough difference that it should be doable.

Re: ticks, I didn't find an easy way to leave them out, so they're still there. I can try cropping the images with ImageMagick before sending them if you want.

**Comment by rcurrie** [comment]
Sounds great - and don't worry about the ticks, probably good to leave them in as we'll have extraneous stuff in the real thing...

**Comment by rcurrie** [comment]
I'm working on a notebook to wrangle these images into a nice tfrecord file that I'll push to S3 so everyone can try various models to train against it in colab etc....

**Comment by rcurrie** [comment]
@atspaeth 49 different labels?

![image](https://user-images.githubusercontent.com/492570/47947974-98119900-dee4-11e8-9550-ec295fd959bc.png)

**Comment by rcurrie** [comment]
@atspaeth @kvoitiuk @ashsrobbins @avivelor I've created a new bucket on PRP as well as namespace all correctly spelled now: braingeneers :)

aws  s3 ls s3://braingeneers/simulation/
2018-11-03 04:39:08    6327552 mltest-cropped-test.tfrecord
2018-11-03 04:39:10   25141200 mltest-cropped-train.tfrecord
2018-11-03 04:39:10   31577599 mltest-cropped.zip

The two tfrecord files are the images with labels stratified by label class. Will work on a simple example of how to access and use these but just wanted to send a quick not about the change in namespace in case you were having trouble before accessing PRP.

**Comment by rcurrie** [comment]
Moving to https://github.com/braingeneers/organoid-net/issues/3
